### PR TITLE
Support for ESLint

### PR DESCRIPTION
**Pull request by dmitmel**
Almost like the `tslint` block, but for JavaScript.

### CODE REVIEW

File: packages/eslint/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# webpack-blocks - TSLint
```
Comment (zcei) [inline packages/eslint/README.md]: Copy 🍝 - needs to be `ESLint` as well 🙂

Comment (dmitmel) [reply packages/eslint/README.md]: Oh, thanks :) These packages are very similar, so I've decided to copy files from the `tslint` block.

Comment (zcei) [reply packages/eslint/README.md]: Sure, that's how I would have done as well 😉

**Comment by andywer** [comment]
Proposal: Let's extract the `tslint` block out of the core webpack-blocks repo into its own and let's put the `eslint` block into a separate repo as well.

We can still link to them in the README here, but trimming down the number of packages here might be a good idea. Thoughts?

@jvanbruegge

**Comment by dmitmel** [comment]
Well, I think these packages must be included in the `webpack-blocks` package because they are used in lots of projects (see [their](https://npm.im/eslint-loader) [stats](https://npm.im/tslint-loader)). But, [your CI can't run all tests](https://travis-ci.org/andywer/webpack-blocks/jobs/344207521#L1132-L1135) because there are too many packages. An ideal solution would be creating an organization and putting all packages into separate repositories.

**Comment by jvanbruegge** [comment]
@andywer
Would be ok with that, just dont forget to make a major version bump on the `webpack-blocks` package. I dont want all the setups suddenly breaking

**Comment by zcei** [comment]
I would really like to see most blocks in their own repo - that's why I initially blocked the [`webpack-blocks` org](https://github.com/webpack-blocks) org.
Though I think especially ESLint (& TSLint ?) should be treated first-class as they're nearly always used.

**Comment by andywer** [comment]
Then maybe let's postpone this decision (extra repo or not) for now 😉

@jvanbruegge That shouldn't be a breaking change. Should have no influence on user's where the sources reside.

@dmitmel Yeah, but the block's are more utility than core functionality and the [download count of the tslint block](https://www.npmjs.com/package/@webpack-blocks/tslint) is quite low.

The CI issue is quite annoying, though... We could try `lerna run --parallel` or `lerna run --stream` to circumvent that issue or try a different monorepo tool.

**Comment by dmitmel** [comment]
@andywer Would you like to add the eslint block or remove the tslint block?

**Comment by andywer** [comment]
@dmitmel Let's add the eslint block for now and decide on the overall strategy afterwards. That discussion shouldn't be a reason to block new features :)

File: packages/eslint/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# @webpack-blocks/eslint - Changelog
+
+## Next release
```
Comment (andywer) [inline packages/eslint/CHANGELOG.md]: In this case we can enter the version number already, since we will publish right away.

**Comment by vlad-zhukov** [comment]
I'd move unpopular and framework-specific blocks out of the main repo (for example `elm`). It will make installs and tests faster (good for both development and CI), but most importantly you can grant write access to people who care about the block, and don't block their development interests.

**Comment by vlad-zhukov** [comment]
@dmitmel Thanks for the PR! Would be great to have a few test cases similar to ones other blocks have.

**Comment by dmitmel** [comment]
@vlad-zhukov I've written some [integration tests](https://github.com/dmitmel/webpack-blocks/blob/feature/eslint-block/packages/eslint/__tests__/eslint.test.js). Thanks for pointing out!

**Comment by andywer** [comment]
Awesome, thanks for your efforts! 😊

**Comment by andywer** [comment]
Published as `@webpack-blocks/eslint@1.0.0-rc` 🚀

**Comment by dmitmel** [comment]
[I've found something wrong](https://github.com/dmitmel/webpack-blocks/commit/71f0940c95d81d43ed11a5fcff3a3f6a739e5068) in the eslint block: it has `@webpack-blocks/core` in its peer dependencies.

**Comment by andywer** [comment]
Might be a copy-pasta mistake. I don't remember with certainty, but I think at some point we added that peer dependency to make explicit that the block only works with webpack-blocks v1 (not v0.x).

It's not critical and the peer dependency is basically true 😉

**Comment by dmitmel** [comment]
@andywer Nice ~hack~ workaround!

**Pull request merged**